doc/input-output-reference/src/overview/group-thermal-zone-description-geometry.tex

```diff
@@ -27,7 +27,9 @@ \subsubsection{Inputs}\label{space-inputs-048}
 
 \paragraph{Field: Floor Area}\label{space-field-floor-area}
 
-Space floor area is used in many places within EnergyPlus. EnergyPlus automatically calculates the space floor area (\SI{}{\area}) from the space geometry given by the surfaces that belong to the space. If this field is 0.0, negative or \textbf{autocalculate} (the default), then the calculated space floor area will be used in subsequent calculations. If this field is positive, then it will be used as the space floor area. If this number differs significantly from the calculated space floor area a warning message will be issued.
+Space floor area is used in many places within EnergyPlus. EnergyPlus automatically calculates the space floor area (\SI{}{\area}) as the total area of surfaces with Surface Type = Floor. If this field is 0.0, negative or \textbf{autocalculate} (the default), then the calculated space floor area will be used in subsequent calculations. If this field is positive, then it will be used as the space floor area. If this number differs significantly from the calculated space floor area a warning message will be issued. 
```

Comment:
What does "significantly" mean in this context? Should there be a percent value or some other quantifiable metric listed here, or maybe note that `DisplayExtraWarnings` can be used to show the issues?

Reply:
I modified this and then got carried away changing the warning messages. They now read:
```
   ** Warning ** GetSurfaceData: Entered Zone Floor Area(s) differ more than 5% from the sum of the Space Floor Area(s).
   **   ~~~   ** ...use Output:Diagnostics,DisplayExtraWarnings; to show more details on individual zones.

   ** Warning ** GetSurfaceData: Entered Floor Area for Zone="SPACE1-1" is 10.2% different from the sum of the Space Floor Area(s).
   **   ~~~   ** Entered Zone Floor Area=90.00, Sum of Space Floor Area(s)=99.16
   **   ~~~   ** Entered Zone Floor Area will be used and Space Floor Area(s) will be adjusted proportionately.

   ** Warning ** GetSurfaceData: Entered Space Floor Area(s) differ more than 5% from calculated Space Floor Area(s).
   **   ~~~   ** ...use Output:Diagnostics,DisplayExtraWarnings; to show more details on individual Spaces.

   ** Warning ** GetSurfaceData: Entered Floor Area for Space="SPACE 1" is 24.0% different from the calculated Floor Area.
   **   ~~~   ** Entered Space Floor Area=80.00, Calculated Space Floor Area=99.16, entered Floor Area will be used.
```

Reply:
Thanks, @mjwitte. Those are perfect. I think this is ready once CI settles down.

Reply:
Grrr. The doc change needs attention. I didn't escape the percent signs. Didn't know I needed to.